Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2863,8 +2763,7 @@ extension BrowserViewController: TabTrayDelegate {
     }
 
     func tabTrayDidAddBookmark(_ tab: Tab) {
-        guard let url = tab.url?.absoluteString, !url.isEmpty else { return }
-        self.addBookmark(tab.tabState)
+        // BRAVE TODO: Not Sure..
```

Comment:
Have you tested if this code gets ever called?

Reply:
At the moment it's never called.. In fact, none of the functions in this delegate get called no matter what, even in the original code.

However, I think it will be called when Peek-Pop is implemented properly because that's where it is called in the code.